l10n_it_codici_carica/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2017 Alessandro Camilli - Openforce
+#
+#    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+
+{
+    'name': 'Codici Carica',
+    'summary':
+        'Aggiunge la tabella dei codici carica da usare nei dichiarativi'
+        ' fiscali italiani',
+    'version': '11.0.1.0.0',
+    'category': 'Account',
+    'author': "Openforce di Camilli Alessandro,"
+        "Odoo Community Association (OCA)",
+    'website': 'http://odoo-italia.net',
+    'license': 'LGPL-3',
+    'depends': [
+        'account',
```

Comment:
Maybe you want to inherit from `account_invoicing`, otherwise your `menuitem` won't be shown (see https://github.com/odoo/odoo/commit/8cd4f48867d27e9046d757e78c9f67d36741bf8b)